### PR TITLE
fix tabIndex navigation arrows

### LINKED ISSUE #2495: Navigating feeds using keyboard arrows
From a user: "In the OPDS Thorium catalog version 3, I cannot move pages using the keyboard (Next button), if I use the mouse click it can work fine."

It is true that for many people, using the left / right arrow key to browse "My books" or an OPDS catalog would be useful.

### PR DESCRIPTION
**Pull request by arthur-lemeur**
Fixes #2495

@llemeurfr, about "I cannot move pages using the keyboard (Next button)" : 
Yes you can do it but it is binded to specific keyboard shortcuts which are by default "Ctrl, SHIFT, ArrowLeft" and "Ctrl, SHIFT, ArrowRight" ("keyboardShortcuts.NavigatePreviousOPDSPage" / "keyboardShortcuts.NavigateNextOPDSPage"

### CODE REVIEW

**Comment by llemeurfr** [comment]
I just tested on the Blbliomedia feed (https://bm.ebibliomedia.ch/v1/home.opds2), which has no "previous page" information: 
Shift + Ctrl + ArrowLeft leads to the **NEXT** page, like Shift + Ctrl + ArrowRight.
To be decided it we consider it a bug (but in this case should Shift + Ctrl + ArrowLeft be inactive or display an error message?) or a feature.

**Comment by panaC** [comment]
> I just tested on the Blbliomedia feed (https://bm.ebibliomedia.ch/v1/home.opds2), which has no "previous page" information: Shift + Ctrl + ArrowLeft leads to the **NEXT** page, like Shift + Ctrl + ArrowRight. To be decided it we consider it a bug (but in this case should Shift + Ctrl + ArrowLeft be inactive or display an error message?) or a feature.

https://github.com/edrlab/thorium-reader/issues/2554

File: src/renderer/library/components/opds/PageNavigation.tsx [hunk 1]
```diff
@@ -97,6 +97,7 @@ class PageNavigation extends React.Component<IProps, undefined> {
                     <button
```
Comment (panaC) [inline src/renderer/library/components/opds/PageNavigation.tsx:97]: We need to remove the button tag, the React Router Link component render an anchor html element and not need to be surrouned by a button tag. The button element doesn't even allow to trigger the button action with space keydown/keyup.

**Comment by panaC** [comment]
I'm closing this PR. Arthur works on a new global OPDS issue PR.

File: src/renderer/library/components/opds/PageNavigation.tsx (hunk 1 above)
Comment (panaC) [reply src/renderer/library/components/opds/PageNavigation.tsx:97]: Okay an anchor wrapped inside a button allows to disable the link anchor.

**Pull request closed without merging**